documentation/schema.yaml

```diff
@@ -780,9 +780,6 @@ referencedSchemas:
         extent:
           type: string
           example: null
-        description:
```

Comment:
It's gone description is?

Reply:
Yes, Rurik removed in previous commit to main https://github.com/BIBSYSDEV/nva-datamodel-java/commit/7f13a7b513b6cf996f277f7ad59bac2e90ec1308